ui/shared/components/panel/search/VariantSearchFormPanels.jsx

```diff
@@ -170,6 +194,13 @@ export const LOCATION_PANEL = {
   helpText: 'Filter by variant location. Entries can be either gene symbols (e.g. CFTR) or intervals in the form <chrom>:<start>-<end> (e.g. 4:6935002-87141054) or separated by tab. Variant entries can be either rsIDs (e.g. rs61753695) or variants in the form <chrom>-<pos>-<ref>-<alt> (e.g. 4-88047328-C-T). Entries can be separated by commas or whitespace.',
 }
 
+export const IN_SILICO_PANEL = {
+  name: 'in_silico',
+  headerProps: { title: 'In Silico Filters' },
+  fields: IN_SILICO_FIELDS,
+  fieldLayout: inSilicoFieldLayout,
```

Comment:
this field needs the following `helpText`:

Filter by in-silico predictors. For numeric filters, any variant with a score greater than or equal to the provided filter value will be returned. For text filters, variants with exactly matching classifications will be returned.